src/components/textarea/TextArea.js

```diff
@@ -333,9 +333,28 @@ export default class TextAreaComponent extends TextFieldComponent {
 
   setReadOnlyValue(value, index) {
     index = index || 0;
+    const brake = '<br>';
```

Comment:
It should be "break" not "brake".

Also, this seems very heavy handed to solve this minor issue. There is not an easier way to solve this?